src/Application/Messenger/Transports.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace MatchBot\Application\Messenger;
+
+use Symfony\Component\Messenger\Bridge\AmazonSqs\Transport\AmazonSqsTransportFactory;
+use Symfony\Component\Messenger\Bridge\Redis\Transport\RedisTransportFactory;
+use Symfony\Component\Messenger\Transport\InMemory\InMemoryTransportFactory;
+use Symfony\Component\Messenger\Transport\Serialization\PhpSerializer;
+use Symfony\Component\Messenger\Transport\TransportFactory;
+use Symfony\Component\Messenger\Transport\TransportInterface;
+
+class Transports
+{
+    public const string TRANSPORT_CLAIMBOT = 'claimbot';
+    public const string TRANSPORT_LOW_PRIORITY = 'low_priority';
+    public const string TRANSPORT_HIGH_PRIORITY = 'high_priority';
```

Comment:
These need to be unique when used as keys for the container, so let's include the context in the strings:

```suggestion
    public const string TRANSPORT_CLAIMBOT = 'transport_claimbot';
    public const string TRANSPORT_LOW_PRIORITY = 'transport_low_priority';
    public const string TRANSPORT_HIGH_PRIORITY = 'transport_high_priority';
```

Will also help with debugging in case we see the string values somewhere.